gsa/src/web/pages/nvts/__tests__/detailspage.js

```diff
@@ -299,102 +122,121 @@ describe('Nvt Detailspage tests', () => {
 
     expect(screen.getAllByTitle('NVT List')[0]).toBeInTheDocument();
     expect(links[1]).toHaveAttribute('href', '/nvts');
-
-    expect(element).toHaveTextContent('ID:12345');
-    expect(element).toHaveTextContent('Mon, Jun 24, 2019 11:55 AM UTC');
-    expect(element).toHaveTextContent('Mon, Jun 24, 2019 10:12 AM UTC');
-    expect(element).toHaveTextContent('Owner:(Global Object)');
-
+    // test title bar
```

Comment:
Calling this title bar might be confusing. Our title bar is the green bar on top of each page. This is the entity info.